book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
Kindly implement a method for adding rentals, something similar to this  :


`    
     def add_rental(person, date)
    Rental.new(date, self, person)
  end`